fhir-tools/src/main/java/com/ibm/fhir/tools/CodeGenerator.java

```diff
@@ -3265,13 +3263,27 @@ public void generateCodeSubtypeClass(String bindingName, String valueSet, String
             ._super(args("builder"))
         .end().newLine();
 
+        cb.javadocStart()
+            .javadoc("Get the value of this " + bindingName + " as an enum constant.")
+            .javadoc("@deprecated replaced by {@link #getValueConstant()}")
+            .javadocEnd();
+        cb.annotation("Deprecated");
```

Comment:
When would we remove it? or leave it until v5?

Reply:
we don't really have a policy for this.  I was just thinking "in a couple releases".  I was thinking to align that with R4B support since that will probably force other model changes as well.

Reply:
Per our conversation in Slack, I'm good to leave it in until 4.9.x (at which team we hopefully have R4B changes in).